images/win/scripts/Tests/Vsix.Tests.ps1

```diff
@@ -0,0 +1,12 @@
+Describe "Vsix" {
+    $toolset = Get-ToolsetContent
+    $requiredVsixs = $toolset.visualStudio.vsix
+
+    $allPackages = (Get-VSSetupInstance | Select-VsSetupInstance -Product *).Packages
+    $testCases = $requiredVsixs | ForEach-Object { @{ VsixId = $_.Id;  AllPackages = $allPackages }}
+
+    It "VSIX Extension <VsixId>" -TestCases $testCases {

```

Comment:
no need in `VSIX` here, we are already in the describe block called `Vsix`

Reply:
VSIX has been removed from It block